input/cql/HIVElements.cql

```diff
@@ -804,6 +789,29 @@ define "Date informed of HIV-positive diagnosis Value":
   return O.value
 /* End of Date informed of HIV-positive diagnosis */
 
+/*
+@activity: HIV.B7 Test for HIV using testing algorithm
+*/
+define "HIV test":
+  [ServiceRequest: Concepts."HIV test type - HIV.B.DE81 Choices"] SR
+  where SR.status in { 'active', 'completed' }
```

Comment:
I'm a bit curious about this mapping:

1. The service request is, as the name implies, a request and not an actual record of a test performed. Unless we're specifically trying to calculate how many tests were _ordered_ its not clear to me that this is semantically the correct resource to use.
2. It makes the "Non-Self HIV test" look (to me) like we're creating a resource that's just their to structurally allow the ServiceRequest to function. HIV self-tests are available on a non-prescription basis, which means that in the normal course of things, you'd only ever have orders for a small sub-set of HIV tests.

The testing algorithm in B7.DT _does_ create ServiceRequests, but the actual _record_ of the test being done, etc. but the final result is always recorded as an algorithm.